packages/features/calendars/DatePicker.tsx

```diff
@@ -332,15 +336,15 @@ const DatePicker = ({
       <div className="mb-1 flex items-center justify-between text-xl">
         <span className="text-default w-1/2 text-base">
           {browsingDate ? (
-            <>
+            <time dateTime={browsingDate.format("YYYY-MM")} data-testid="selected-month-label">
```

Comment:
Add a time component here for both HTML semantics and making it easier to test in testcases without relying on i18n translations (potentially)

testid added for selection.